Makefile

```diff
@@ -121,48 +134,68 @@ vald/sha/print:
 
 .PHONY: vald/sha/update
 ## update VALD_SHA value
-vald/sha/update: vald
-	(cd vald; git rev-parse HEAD | tr -d '\n' > ../$(VALD_SHA))
+vald/sha/update: $(VALD_DIR)
+	(cd $(VALD_DIR); git rev-parse HEAD | tr -d '\n' > ../$(VALD_SHA))
 
-.PHONY: vald/client/python/version/print
+.PHONY: vald/client/version/print
 ## print VALD_CLIENT_PYTHON_VERSION value
-vald/client/python/version/print:
+vald/client/version/print:
 	@cat $(VALD_CLIENT_PYTHON_VERSION)
 
-.PHONY: vald/client/python/version/update
+.PHONY: vald/client/version/update
 ## update VALD_CLIENT_PYTHON_VERSION value
-vald/client/python/version/update: vald
+vald/client/version/update: $(VALD_DIR)
 	(vald_version=`cat $(VALD_DIR)/versions/VALD_VERSION | sed -e 's/^v//'`; \
-	    client_version=`cat $(VALD_CLIENT_PYTHON_VERSION)`; \
-	    major=$${client_version%%.*}; client_version="$${client_version#*.}"; \
-	    minor=$${client_version%%.*}; client_version="$${client_version#*.}"; \
-	    patch=$${client_version%%.*}; client_version="$${client_version#*.}"; \
-	    if [ "$${vald_version}" = "$${major}.$${minor}.$${patch}" ]; then \
-	        if [ "$${patch}" = "$${client_version}" ]; then \
-	            new_version="$${major}.$${minor}.$${patch}.post1"; \
-	        else \
-	            rev="$${client_version#post}"; \
-	            rev=$$((rev+1)); \
-	            new_version="$${major}.$${minor}.$${patch}.post$${rev}"; \
-	        fi; \
-	    else \
-	        new_version="$${vald_version}"; \
-	    fi; \
-	    echo "VALD_VERSION: $${vald_version}, NEW_CLIENT_VERSION: $${new_version}"; \
-	    echo "$${new_version}" > VALD_CLIENT_PYTHON_VERSION)
+	    echo "VALD_VERSION: $${vald_version}"; \
+	    echo "$${vald_version}" > VALD_CLIENT_PYTHON_VERSION)
 	sed -i -e "s/^version = .*\$$/version = `cat VALD_CLIENT_PYTHON_VERSION`/" setup.cfg
 
-.PHONY: proto/deps
+.PHONY: test
+## Execute test
+test: $(TEST_DATASET_PATH)
+	python src/test.py
+
+$(TEST_DATASET_PATH):
+	curl -L https://raw.githubusercontent.com/rinx/word2vecjson/master/data/wordvecs1000.json -o $(TEST_DATASET_PATH)
+
+.PHONY: ci/deps/install
+## install deps for CI environment
+ci/deps/install: proto/deps/install
+	sudo apt-get update -y && sudo apt-get install -y \
+		python3-setuptools \
+		libprotobuf-dev \
+		libprotoc-dev \
+		protobuf-compiler
+	pip3 install grpcio-tools
```

Comment:
How do you feel about this suggestion?

```suggestion
PIP = pip3

.PHONY: ci/deps/install
## install deps for CI environment
ci/deps/install: proto/deps/install
	sudo apt-get update -y && sudo apt-get install -y \
		python3-setuptools \
		libprotobuf-dev \
		libprotoc-dev \
		protobuf-compiler
	$(PIP) install grpcio-tools
```

Reply:
Thank you for your suggestion 🙏  It looks good, so I apply your suggestion!